.github/workflows/android-env.yml

```diff
@@ -27,11 +25,22 @@ jobs:
     name: 'Build'
     runs-on: ubuntu-latest
     steps:
-      - name: "Build:checkout"
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```

Comment:
Can you please update to actions/checkout@v3 which uses Node.js 16 instead of Node.js 12 which has been deprecated since April 2022